public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -130,6 +162,17 @@ export class ApiKeysPage extends PureComponent<Props, any> {
     );
   }
 
+  formatDate(date, format?) {
+    if (!date) {
+      return '';
+    }
+    date = isDateTime(date) ? date : dateTime(date);
+    format = format || 'YYYY-MM-DD HH:mm:ss';
+    const timezone = store.getState().user.timeZone;
```

Comment:
There's a selector called `getTimeZone` [here](https://github.com/grafana/grafana/blob/master/public/app/features/profile/state/selectors.ts#L3) that's suggested to be used instead of traversing the state tree in a component.